pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -623,6 +623,11 @@ func (nc *Controller) doNoExecuteTaintingPass(ctx context.Context) {
 				return false, 50 * time.Millisecond
 			}
 			_, condition := controllerutil.GetNodeCondition(&node.Status, v1.NodeReady)
```

Comment:
It looks like first return value of the `controllerutil.GetNodeCondition` is not used anywhere. Would it make sense to simplify it to return one value and check that the value is checked everywhere where `controllerutil.GetNodeCondition` is called?

Reply:
> It looks like first return value of the `controllerutil.GetNodeCondition` is not used anywhere. Would it make sense to simplify it to return one value and check that the value is checked everywhere where `controllerutil.GetNodeCondition` is called?

there are several functions have called controllerutil.GetNodeCondition, but only this one does not check if the result value is nil.

Reply:
Sorry for not being clear enough. controllerutil.GetNodeCondition has 2 return values. First one is not used anywhere, so can be dropped. This should simplify both the function and the caller's code.

Reply:
Thank you for your reminder, I have committed again, please review.